<launch>

  <!-- <node pkg="rosmsg_tcp_bridge" name="drone_$(arg drone_id)_bridge_node" type="bridge_node" output="screen" launch-prefix="valgrind" > -->
  <node pkg="rosmsg_tcp_bridge" name="drone_bridge_node" type="bridge_node" output="screen" >
    <remap from="~my_odom" to="/vins_estimator/imu_propagate"/>   
    <param name="broadcast_ip" value="192.168.1.255" type="string"/>
    <param name="odom_max_freq" value="10"/>
    <param name="formation_scale" value="1.0"/>
  </node>

</launch>
